addons/l10n_id/models/res_bank.py

```diff
@@ -91,6 +91,9 @@ def _get_qr_vals(self, qr_method, amount, currency, debtor_partner, free_communi
                 "cliTrxAmount": int(amount)
             }
             response = _l10n_id_make_qris_request('show_qris.php', params)
+            if response.get("status") == "failed":
+                msg = response.get("data")
+                raise ValidationError(_("%s", msg))
```

Comment:
```suggestion
                raise ValidationError(_("QRIS Error: %s", msg))
```
I would also probably add some kind of lead to say which app is responsible for the issue :thinking: @vin-odoo what do you think?

Reply:
I don't think it's matters as long as the error is clear, the user should know the issue based on what they are doing at the moment IMO